### PR TITLE
Build for ElectronJS

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

**Comment by lissyx** [comment]
We can build for a lot of ElectronJS flavors, but I suggest we stick with testing at least linux/macOS/windows on amd64. We can try and add more testing later, depending on people's usecases and feedback.

File: tc-tests-utils.sh [hunk 1]
```diff
@@ -1110,6 +1132,17 @@ do_deepspeech_nodejs_build()
...
+    EXTRA_CFLAGS="${EXTRA_LOCAL_CFLAGS}" EXTRA_LDFLAGS="${EXTRA_LOCAL_LDFLAGS}" EXTRA_LIBS="${EXTRA_LOCAL_LIBS}" make -C native_client/javascript \
+      TARGET=${SYSTEM_TARGET} \
+      RASPBIAN=${SYSTEM_RASPBIAN} \
+      TFDIR=${DS_TFDIR} \
+      NODE_ABI_TARGET=--target=$electron \
+      NODE_DIST_URL=--disturl=https://atom.io/download/electron \
+      NODE_RUNTIME=--runtime=electron \
+      clean node-wrapper
+  done;
+
```
Comment (reuben) [inline tc-tests-utils.sh]: Is this roughly doubling the size of the Node package as well as the time to build it? Could we separate Node.JS and Electron packages, or does that not make sense in the Node ecosystem?

Comment (lissyx) [reply tc-tests-utils.sh]: It seems that it does not make sense in the ecosystem, but it's not really doubling, we just add a few `deepspeech.node` binaries, ranging from 50k to 100k. All docs I found would pull from `npmjs` for ElectronJS. And build time does increase, but it's marginal, takes < 30 secs for each ElectronJS instance. We would win much more build time by doing the heavy lifting of producing ready-to-use Docker images.

Comment (reuben) [reply tc-tests-utils.sh]: Understood. My review+ remains :)

**Pull request merged**